tests/testdata/inputs/alternate-configs/other.yaml

```diff
@@ -0,0 +1,15 @@
+# Copyright 2019 Google LLC
```

Comment:
Now that I know about pyfakefs, I filed a clean-up [issue](https://github.com/googleapis/sample-tester/issues/107) to use it in place of these testdata files.